core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -3344,50 +3344,109 @@ public String toString() {
+          .setDescription("The hostname of the job master web server.")
           .setDefaultValue("${alluxio.job.master.hostname}")
           .build();
   public static final PropertyKey JOB_MASTER_WEB_PORT =
-      new Builder(Name.JOB_MASTER_WEB_PORT).setDefaultValue(20002).build();
+      new Builder(Name.JOB_MASTER_WEB_PORT)
+          .setDescription("The port the job master web server uses.")
+          .setDefaultValue(20002)
+          .build();
   public static final PropertyKey JOB_WORKER_BIND_HOST =
-      new Builder(Name.JOB_WORKER_BIND_HOST).setDefaultValue("0.0.0.0").build();
+      new Builder(Name.JOB_WORKER_BIND_HOST)
+          .setDescription("The host that the Alluxio job worker will bind to.")
+          .setDefaultValue("0.0.0.0")
+          .build();
   public static final PropertyKey JOB_WORKER_DATA_PORT =
-      new Builder(Name.JOB_WORKER_DATA_PORT).setDefaultValue(30002).build();
-  public static final PropertyKey JOB_WORKER_HOSTNAME = new Builder(Name.JOB_WORKER_HOSTNAME)
-      .setDescription("The hostname of Alluxio job worker.")
-      .setScope(Scope.WORKER)
-      .build();
+      new Builder(Name.JOB_WORKER_DATA_PORT)
+          .setDescription("The port the Alluxio Job worker uses to send data.")
+          .setDefaultValue(30002)
+          .build();
+  public static final PropertyKey JOB_WORKER_HOSTNAME =
+      new Builder(Name.JOB_WORKER_HOSTNAME)
+          .setDescription("The hostname of the Alluxio job worker.")
+          .setScope(Scope.WORKER)
+          .build();
   public static final PropertyKey JOB_WORKER_RPC_PORT =
-      new Builder(Name.JOB_WORKER_RPC_PORT).setDefaultValue(30001).build();
+      new Builder(Name.JOB_WORKER_RPC_PORT)
+          .setDescription("The port the job worker uses to send RPCs")
+          .setDefaultValue(30001)
+          .build();
   public static final PropertyKey JOB_WORKER_WEB_BIND_HOST =
-      new Builder(Name.JOB_WORKER_WEB_BIND_HOST).setDefaultValue("0.0.0.0").build();
+      new Builder(Name.JOB_WORKER_WEB_BIND_HOST)
+          .setDescription("The host the job worker web server binds to.")
+          .setDefaultValue("0.0.0.0")
+          .build();
   public static final PropertyKey JOB_WORKER_WEB_PORT =
-      new Builder(Name.JOB_WORKER_WEB_PORT).setDefaultValue(30003).build();
-
+      new Builder(Name.JOB_WORKER_WEB_PORT)
+          .setDescription("The port the Alluxio job worker web server uses.")
+          .setDefaultValue(30003)
+          .build();
   public static final PropertyKey JOB_MASTER_RPC_ADDRESSES =
-      new Builder(Name.JOB_MASTER_RPC_ADDRESSES).build();
+      new Builder(Name.JOB_MASTER_RPC_ADDRESSES)
+          .setDescription("The list of RPC addresses to use for the job service configured in "
+              + "non-zookeeper HA mode. If this property is not specifically defined, it will "
+              + "first fall back to using the alluxio.master.rpc.addresses replacing those "
```

Comment:
I'd consider "alluxio.master.rpc.addresses" singular since it's a property key, not the addresses themselves

I think that just means changing "using the alluxio.master.rpc.addresses" to "using alluxio.master.rpc.addresses"